### PR TITLE
Feature/improve plotmanager2 jonybang

### PR DESCRIPTION
**Pull request by Jonybang**

### CODE REVIEW

File: contracts/PlotManager.sol [hunk 1]
```diff
@@ -254,6 +277,10 @@ contract PlotManager is Initializable, Ownable {
 
     // TODO: implement directions
     splitMerge.removeGeohashesFromPackage(a.packageTokenId, _geohashes, _directions1, _directions2);
+
+    if(splitMerge.getPackageCount(a.packageTokenId) == 0 && a.status == ApplicationStatuses.NEW) {
```
Comment (chebykin) [inline contracts/PlotManager.sol]: should we match rejected as well?

File: contracts/SplitMerge.sol [hunk 2]
```diff
@@ -138,6 +138,14 @@ contract SplitMerge is Initializable, Ownable {
     return packageToGeohashes[_packageToken];
   }
 
+  function getPackageCount(uint256 _packageToken) public view returns (uint256) {
+    return packageGeohashesCount[_packageToken];
+  }
```
Comment (chebykin) [inline contracts/SplitMerge.sol]: You should be able to access geohashesCount for a packageToken both from web3 and another contract since it's public.

Comment (chebykin) [reply contracts/SplitMerge.sol]: Just use a field name as a method and a tokenId as an argument with web3 like:
```javascript
await contractInstance.methods.packageGeohashesCount(tokenId).send();
```

File: test/integration/testPlotManager.js [hunk 3]
```diff
@@ -582,6 +582,28 @@ contract('PlotManager', ([coreTeam, galtSpaceOrg, alice, bob, charlie]) => {
...
+        await this.plotManager.removeGeohashesFromApplication(this.aId, geohashesToRemove, [], [], {
+          from: alice
+        });
+
+        res = await this.plotManagerWeb3.methods.getApplicationById(this.aId).call();
+        assert.equal(res.status, ApplicationStatuses.DISASSEMBLED);
+
+        res = await this.splitMerge.packageGeohashesCount(res.packageTokenId);
+        assert.equal(res, 0);
+      });
```
Comment (chebykin) [inline test/integration/testPlotManager.js]: It looks like I forgot about DISASSEBLED test 😱. Thanks!

File: contracts/SplitMerge.sol (hunk 2 above)
Comment (Jonybang) [reply contracts/SplitMerge.sol]: @chebykin i was try to access geohashesCount mapping from another contract, but received error. I think no way to do this

File: contracts/PlotManager.sol (hunk 1 above)
Comment (Jonybang) [reply contracts/PlotManager.sol]: Documentation says: "Validator can receive a reward if rejected application have 0 geohashes in pack(disassembled)"

**Pull request merged**